source/_posts/2015-10-09-the-slack-users-code.markdown

```diff
@@ -0,0 +1,258 @@
+---
+layout: post
+title: "The Slack Users' Code"
+date: 2015-10-09 22:30:00 -0500
+comments: true
+categories:
+---
+Without a code, [Slack](https://slack.com/) would be chaos. These are my ideas
+about what makes a good Slack team and Slack citizen. These are for _work Slack
+teams_. If you have a social Slack team set up, you do you because the
+expectations, goals and social forces are likely to be very different.
+
+
+## Yourself
+
+If you work in a widely distributed business unit, you should be interacting
+with people across the world from time to time. These are people who do not know
+you by sight. These four guidelines are to help those people know who you are
+and maybe have a chance of recognizing you from #watercooler when they meet you
+in person.
+
+* You should set your first and last name in your
+  [Slack Profile](https://my.slack.com/account/profile).
+* If your full name is First Middle Last, your user name should be firstlast,
+  flast, fmlast or something similar. Change it in
+  [the username settings](https://my.slack.com/account/settings#username).
+* Your avatar should be your face. Ideally, it should be big enough that it
+  takes up most of the avatar space. It should be readable and recognizable at a
+  small size.
+* Put your job title _and your team name_ in the "What I Do" section of
+  [your Slack Profile](https://my.slack.com/account/profile). If you're an
+  administrator, consider using [Slack's custom profiles](http://slackhq.com/post/132116511325/customprofiles)
+  to make a special space for this information.
+
+
+## Channels
+
+Channels, especially, are important to name in a consistent and predictable way.
+You want channels to be discoverable and obvious so that if someone is trying to
```

Comment:
Alternate, more direct: “…so that it’s easier for people to find channels to ask questions to start a discussion.”